test/support/exercise_test_case.ex

```diff
@@ -147,4 +151,21 @@ defmodule ElixirAnalyzer.ExerciseTestCase do
   def assert_comments(_, _, _) do
     :noop
   end
+
+  def find_exemplar_code(exercise_test_module) do
+    with {slug, _module} <-
+           Enum.find(@exercise_config, fn
+             {_slug, %{analyzer_module: ^exercise_test_module}} -> true
+             _ -> false
```

Comment:
This is not super efficient because I'm going through the whole list of modules to find the original slug...

Reply:
I guess this for our own unit testing purposes that doesn't matter much.